src/btree/bt_split.c

```diff
@@ -405,8 +383,7 @@ __split_deepen(WT_SESSION_IMPL *session, WT_PAGE *parent)
 	 * Create N children, unless we are dealing with a large page without
 	 * many entries, in which case split into the minimum number of pages.
 	 */
-	children = WT_MAX(pindex->entries / __split_deepen_per_child,
-	    __split_deepen_min_child);
+	children = pindex->entries / __split_deepen_per_child;
```

Comment:
We're not dividing by `__split_deepen_min_child`, we're dividing by a different constant.  It happens to be smaller in the values you've chosen, but the previous code didn't rely on that.